docs/tutorial/online-offline-events.md

```diff
@@ -1,7 +1,12 @@
 # Online/Offline Event Detection
 
-Online and offline event detection can be implemented in the renderer process
-using standard HTML5 APIs, as shown in the following example.
+[Online and offline event](https://developer.mozilla.org/en-US/docs/Online_and_offline_events) detection can be implemented in the renderer process using [NavigatorOnLine](http://html5index.org/Offline%20-%20NavigatorOnLine.html) attribute, part of standard HTML5 API.
+The navigator.onLine attribute returns `false` if the user agent will not contact the network when the user follows links or when a script requests a remote page (or knows that such an attempt would fail) i.e. definitely offline (disconnected from the network) and must return `true` otherwise.
```

Comment:
I think we can make this simpler to understand.

```markdown
The navigator.onLine attribute returns `false` if any network requests are guaranteed to fail.  This normally occurs when the device is disconnected from the network.  It will return `true` in all other cases.
```

Reply:
Will do.